codes/quantum/qubits/t-designs.yml

```diff
@@ -0,0 +1,91 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+# Use UTF-8 unicode encoding
+# AMS-TeX commands are rendered inside \( ... \) using MathJaX.
+# Allowed external bibliographic references are
+#    \cite{arXiv:#.#} or \cite{arXiv:quant-ph/#},
+#    \cite{doi:#}, or, as a last resort
+#    \cite{manual:{(enter citation line incl. author and year here)}}
+# External websites such as code tables, coding theory packages, github pages linked as
+#    \url{https://example.com/example}
+#    \href{https://example.com/example}{link text}
+# Internal references to codes are 
+#    \hyperref[code:code_id]{link text}
+# Delete instructional comments when submitting
+
+# code id, physical, logical are all lower case
+code_id: t-designs
+physical: qubits
+logical: qubits
+
+# Apostrophes are denoted by two apostrophe characters, i.e., ''
+# Code title is singular
+name: 'T-design code'
+
+# First reference with the code (optional).
+introduced: 'See Application III in \cite{doi:10.1007/s00220-016-2706-8}'
+
+# Anything applicable to a larger parent set of codes (see below) should go in
+# that entry instead of here.
+description: 'A state has topological quantum order if there exists an orthogonal state which cannot be distinguished from it by local measurements. In this code, the code states are a pair of approximately topologically ordered states produced by starting with any two orthogonal states on \(n\) qubits and acting with random local circuits of depth \(\leq 10^9 n\).'
```

Comment:
Let's avoid the use of "toplogical" order as it has certain connotations that do not apply.  Use the term "two states are locally indistinguishable ...".  Can add comment that this is analogous to the definition of topological order from CMP.

Reply:
The purpose of the description entry is to give a mental picture of the code. For this, maybe the following applies... "Code states can be generated by taking a random unitary circuit with (specify what kind of gates... geometrically local? just local?) of depth (constant? linear? log? poly?) and applying it to vacuum."